src/main/resources/warnings.json

```diff
@@ -0,0 +1,107 @@
+[
+  {
+    "id": "SECURITY-208",
+    "type": "plugin",
+    "name": "google-login",
+    "message": "Authentication bypass vulnerability",
+    "uri": "https://wiki.jenkins-ci.org/display/SECURITY/Jenkins+Security+Advisory+2015-10-12",
+    "versions": [
+      {
+        "lastVersion": "1.1",
+        "pattern": "1[.][01](|[.-].*)"
+      }
+    ]
+  },
+  {
+    "id": "SECURITY-136",
+    "type": "plugin",
+    "name": "extra-columns",
+    "message": "Stored XSS vulnerability",
+    "uri": "https://wiki.jenkins-ci.org/display/SECURITY/Jenkins+Security+Advisory+2016-04-11",
+    "versions": [
+      {
+        "lastVersion": "1.16",
+        "pattern": "1[.](\\d|1[0123456])(|[.-].*)"
```

Comment:
It would not match the expression as it only expects two digits after the period, IOW it would behave as expected.